buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClusterConfiguration.java

```diff
@@ -86,6 +88,8 @@
 
     void restart();
 
+    void goToNextVersion();
```

Comment:
Maybe `upgradeToNextVersion()`?

Reply:
It could technically be a downgrade too.

Reply:
Got it.